src/view-components/account-header/index.ts

```diff
@@ -0,0 +1,7 @@
+import './account-header.css';
+import Handlebars from 'handlebars';
+import { accountHeaderTemplate } from './account-header.tmpl';
+
+export function registerAccountHeaderComponent() {
+  Handlebars.registerPartial('app-account-header', accountHeaderTemplate);
+}
```

Comment:
Вот если оставить так, то компонент который будет импортить этот компонент сможет использовать этот паршиал в своем шаблоне. То есть грубо говоря для регистрации достаточно импорта js-файла с зарегистрированным шаблоном. Например `import 'path-to-js-file/account-header'`
```suggestion
Handlebars.registerPartial('app-account-header', accountHeaderTemplate);
```

Reply:
я бы предпочел отказаться от импортов, которые ничего не возвращают, а сложил подключение всех partials для Нandlebar в одно место, и подключил при старте приложения. В любом случае, partials - legacy, и в рамках 3-4 спринтов я от них откажусь в пользу нормальных компонентов, которые будут импортироваться только по необходимости